### PR TITLE
PS-8659 Add a note that percona-server-shared-compat package is not provided with RHEL9 (5.7)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/installation/yum_repo.md

### CODE REVIEW

File: docs/installation/yum_repo.md [hunk 1]
```diff
@@ -19,19 +19,21 @@ Specific information on the supported platforms, products, and versions are desc
...
+* The `Percona-Server-client-57` package contains the command line client.
 
-The `Percona-Server-devel-57` package contains the header files needed to compile software using the client library.
+* The `Percona-Server-devel-57` package contains the header files needed to compile software using the client library.
 
-The `Percona-Server-shared-57` package includes the client shared library.
+* The `Percona-Server-shared-57` package includes the client shared library.
 
-The `Percona-Server-shared-compat` package includes shared libraries for software compiled against old versions of the client library. Following libraries are included in this package: `libmysqlclient.so.12`, `libmysqlclient.so.14`, `libmysqlclient.so.15`, `libmysqlclient.so.16`, and `libmysqlclient.so.18`.
+* The `Percona-Server-shared-compat` package includes shared libraries for software compiled against old versions of the client library. Following libraries are included in this package: `libmysqlclient.so.12`, `libmysqlclient.so.14`, `libmysqlclient.so.15`, `libmysqlclient.so.16`, and `libmysqlclient.so.18`.
```
Comment (patrickbirch) [inline docs/installation/yum_repo.md]: against older versions 
The following libraries

File: docs/installation/yum_repo.md [hunk 2]
```diff
@@ -19,19 +19,21 @@ Specific information on the supported platforms, products, and versions are desc
...
-The `Percona-Server-shared-57` package includes the client shared library.
+* The `Percona-Server-shared-57` package includes the client shared library.
 
-The `Percona-Server-shared-compat` package includes shared libraries for software compiled against old versions of the client library. Following libraries are included in this package: `libmysqlclient.so.12`, `libmysqlclient.so.14`, `libmysqlclient.so.15`, `libmysqlclient.so.16`, and `libmysqlclient.so.18`.
+* The `Percona-Server-shared-compat` package includes shared libraries for software compiled against old versions of the client library. Following libraries are included in this package: `libmysqlclient.so.12`, `libmysqlclient.so.14`, `libmysqlclient.so.15`, `libmysqlclient.so.16`, and `libmysqlclient.so.18`. 
 
-The `Percona-Server-test-57` package includes the test suite for Percona Server for MySQL.
+    This package is not included in RHEL9 downloads.
+
+* The `Percona-Server-test-57` package includes the test suite for Percona Server for MySQL.
```
Comment (patrickbirch) [inline docs/installation/yum_repo.md:36]: Red Hat Enterprise Linux 9 and derivatives

File: docs/installation/yum_repo.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/installation/yum_repo.md]: updated

File: docs/installation/yum_repo.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/installation/yum_repo.md:36]: Updated

**Review by patrickbirch — APPROVED** [review]
Good

**Pull request merged**